skorch/callbacks/lr_scheduler.py

```diff
@@ -66,11 +72,21 @@ def __init__(self,
                  policy='WarmRestartLR',
                  monitor='train_loss',
                  event_name="event_lr",
+                 step_every='epoch',
                  **kwargs):
         self.policy = policy
         self.monitor = monitor
         self.event_name = event_name
+        self.step_every = step_every
         vars(self).update(kwargs)
+        if policy == TorchCyclicLR or policy == "TorchCyclicLR":
+            warnings.warn(
+                "The LRScheduler callback makes a step "
+                "every epoch by default from now on. To have the cyclic lr "
+                "scheduler update every batch, "
+                "set step_every='batch'",
+                DeprecationWarning,
```

Comment:
FutureWarning shows by default to users.

```suggestion
                FutureWarning,
```